lib/net.js

```diff
@@ -510,6 +511,13 @@ function Socket(options) {
   // Used after `.destroy()`
   this[kBytesRead] = 0;
   this[kBytesWritten] = 0;
+  if (options.blockList) {
+    if (options.blockList instanceof module.exports.BlockList) {
```

Comment:
TODO for later... we should have a more efficient `isBlockList(...)` API

Reply:
This has landed :-)

Reply:
Cool ! Have updated the code.